src/elements/common/messages.js

```diff
@@ -788,10 +788,15 @@ const messages: { [string]: MessageDescriptor } = defineMessages({
         defaultMessage: 'Completed',
         description: 'Title for checkmark icon indicating someone completed a task',
     },
+    appActivityDeleteErrorMessage: {
+        id: 'be.appActivityDeleteErrorMessage',
+        description: 'Error message when an app activity deletion fails',
+        defaultMessage: 'There was an error deleting this app activity item.',
```

Comment:
Are we displaying the string "app activity" anywhere? A user might confused as to what an app activity item is vs. just saying item.